### ISSUE TITLE
Java sdk 11[BUG]

### ISSUE THREAD

**Issue by sirbum69** [Stephan-S/FS19_AutoDrive#1337]
**AutoDrive Version:** 1.1.x.x   
**Release or Custom Zip:** Release  
**Map (If applicable):** All  
**Vehicle / Type (If applicable):**   
**Multiplayer or Singleplayer:** Singleplayer  

**Describe the bug**  

**If applicable - Steps to reproduce the behavior**  

**Screenshots**  

the editor requires Java SDk 11 or greater but this is not a free download. you have to pay for Java SDK now. and regular Java is only at version 8. Why would you create a program people can not use without having to pay for Java.

**Comment by raupach** [comment]
[Java is Still Free](https://docs.google.com/document/d/1nFGazvrCvHMZJgFstlbzoHjpAVwv5DEdnaBr_5pKuHo/preview)

**Comment by raupach** [comment]
tl;dr
Java is still free for personal use. I guess that should apply to everyone here, or do you run a data center?

**Comment by sirbum69** [comment]
It is for the normal java yes, but the normal one is only at 8 and the program when I try to run it says I need 11 because it was made with a newer version then what I have. So I went out to look and the only one that is 11 is the SDk that is not free, I had to sign into an account in order to get it.

**Comment by ShioR** [comment]
> The new license permits certain uses, such as personal use and development use, at no cost

https://www.oracle.com/java/technologies/javase-jdk11-downloads.html

> Oracle is keeping Java free and open, and providing stability, performance and security updates to the current version, at no cost. 

https://blogs.oracle.com/java-platform-group/oracle-java-se-releases-faq

Still seems free to download JDK 11 for personal use to me? Having a free account somewhere doesn't make it paid merely because an account is needed, I'm sure they won't mind if you use fake details and a temporary email address.

**Comment by raupach** [comment]
It is mentioned that **at least** Java version 11 is required. Of course, a current version 14 will also work, and it has always been the case that an Oracle account was required for older versions. But the current Java version can always be downloaded **WITHOUT** an account.

It is also a good option to use the OpenJDK (I use that too). Here are all downloads: 
https://jdk.java.net/archive/
